manual/manual/tutorials/objectexamples.etex

```diff
@@ -2,15 +2,20 @@
 \label{c:objectexamples}
 \pdfchapterfold{-15}{Tutorial: Objects in OCaml}
 %HEVEA\cutname{objectexamples.html}
-{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
+{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
 
 \bigskip
 
 \noindent This chapter gives an overview of the object-oriented features of
-OCaml. Note that the relation between object, class and type
-in OCaml is very different from that in mainstream
-object-oriented languages like Java or C++, so that you should not
-assume that similar keywords mean the same thing.
+OCaml.
+
+Note that the relationship between object, class and type in OCaml is different than
+in Java and C++, so you shouldn't assume that similar keywords mean the same thing.
```

Comment:
"~is different than~ **differs from** in Java and C++, so ~you~ **one** shouldn't assume that similar keywords ~mean the same thing~ **have the same meaning**."? (I realise the last part is not your text originally!)

Reply:
These seem rather subjective--no single right/best way to word this.

As I see it: "... in OCaml is different than in Java and C++'  has simple parallel construction with the 2 uses of "in \<languages>".  The bold part in your suggested "Note that the relationship between object, class and type in OCaml **differs from in** Java and C++" is itself awkward; "differs from that in" is more specific but feels pedantic.

"you" vs. "one": the document uses "you" and "your" in many places.  Less formal and friendlier, seems OK to me.

"have the same meaning" vs "mean the same thing".  The latter is a more active construction, the former is sort of a passive voice.